app/controllers/admin/events_controller.rb

```diff
@@ -150,6 +149,34 @@ def toggle_attendance
       end
     end
 
+    ##
+    # Received a file from user
+    # Reads file and creates commercial for event
+    # File content example:
+    # EventID:MyURL
+    def mass_commercials
+      if params[:file]
+        errors = []
+
+        params[:file][:file].read.each_line do |line|
```

Comment:
First [:file] is from the `=semantic_form_for :file` and the second is from the `= f.input :file` but I'll change that to
`= semantic_form_for ''` and simply have `params[:file]`